Makefile

```
@@ -51,3 +51,11 @@ bundle-pre:
 	-$(YARN) unlink @patternslib/pat-code-editor
 	-$(YARN) unlink @patternslib/patternslib
 	$(YARN) install --force
+
+.PHONY: storybook
+storybook: ## Run Storybook in dev mode
```

This avoids the issue reported by @acsr.

```suggestion
storybook: install ## Run Storybook in dev mode
```